ratelimit.go

```diff
@@ -24,7 +24,8 @@ import (
 	"sync"
 	"time"
 
-	"go.uber.org/ratelimit/internal/clock"
+	"context"
```

Comment:
nit: no blank between stdlib imports

Reply:
Done

Reply:
still see a blank between "time" and "context"